pages/common/pg_isready.md

```diff
@@ -0,0 +1,12 @@
+# pg_isready
+
+> Check the connection status of a PostgreSQL server.
+> More information: <https://www.postgresql.org/docs/current/app-pg-isready.html>.
+
+- Check connection with a specific hostname and port:
+
+`pg_isready -h {{hostname}} -p {{port}}`
+
+- Run connection status quietly:
+
+`pg_isready --quiet`
```

Comment:
Is `--host` required?

Reply:
host is not mandatory. default host is `localhost`.